frontend-react/src/hooks/UseValueSets.ts

```diff
@@ -32,7 +32,7 @@ const findTableMetaByName = (
     );
 
     if (!filteredBody.length) {
-        console.log("Unable to find metadata for lookup table: ", tableName);
+        console.error("Unable to find metadata for lookup table: ", tableName);
```

Comment:
maybe console.info here instead of error.

Reply:
So change all the more informal `console.log` we have around the production codebase to `console.info`?

Reply:
No, i'm just suggesting for this specific console to be console.info since it doesn't seem like an error.

Reply:
Sounds good.